CONTRIBUTING.md

```diff
@@ -15,85 +15,85 @@ product better.
-   * **User documentation** is built from several sources and published at 
-     [docs.openvino.ai](https://docs.openvino.ai/), which is the recommended place for reading 
-     these documents. Use the files maintained in this repository only for editing purposes. 
+   * **User documentation** is built from several sources and published at
+     [docs.openvino.ai](https://docs.openvino.ai/), which is the recommended place for reading
+     these documents. Use the files maintained in this repository only for editing purposes.
 
-   * The easiest way to help with documentation is to review it and provide feedback on the 
-     existing articles. Whether you notice a mistake, see the possibility of improving the text, 
-     or think more information should be added, you can reach out to any of the documentation 
+   * The easiest way to help with documentation is to review it and provide feedback on the
+     existing articles. Whether you notice a mistake, see the possibility of improving the text,
+     or think more information should be added, you can reach out to any of the documentation
      contributors to discuss the potential changes.
-     
+
      You can also create a Pull Request directly, following the [editor's guide](./CONTRIBUTING_DOCS.md).
 
 
 ### Promote and Support OpenVINO
 
-   * **Popularize OpenVINO**  
-     Articles, tutorials, blog posts, demos, videos, and any other involvement 
-     in the OpenVINO community is always a welcome contribution. If you discuss 
+   * **Popularize OpenVINO**
+     Articles, tutorials, blog posts, demos, videos, and any other involvement
+     in the OpenVINO community is always a welcome contribution. If you discuss
      or present OpenVINO on various social platforms, you are raising awareness
      of the product among A.I. enthusiasts and enabling other people to discover
      the toolkit. Feel free to reach out to OpenVINO developers if you need help
      with making such community-based content.
 
-   * **Help Other Community Members**  
-     If you are an experienced OpenVINO user and want to help, you can always 
-     share your expertise with the community. Check GitHub Discussions and 
-     Issues to see if you can help someone. 
+   * **Help Other Community Members**
+     If you are an experienced OpenVINO user and want to help, you can always
+     share your expertise with the community. Check GitHub Discussions and
+     Issues to see if you can help someone.
 
 ## Technical guide
 
 This section lists all the necessary steps required to set up your environment, build OpenVINO locally, and run tests for specific components. It's a perfect place to start when you have just picked a Good First Issue and are wondering how to start working on it.
 
 Keep in mind that we are here to help - **do not hesitate to ask the development team if something is not clear**. Such questions allow us to keep improving our documentation.
 
-### 1. Prerequisites 
+### 1. Prerequisites
```

Comment:
- [Model Optimizer](https://github.com/openvinotoolkit/openvino/tree/master/tools/mo) - Should be replaced with OCV